package.json

```diff
@@ -28,6 +28,12 @@
   "homepage": "https://github.com/wikimedia/wikimedia-wordpress-annual-report-plugin#readme",
   "devDependencies": {
     "@wordpress/scripts": "^27.4.0",
+    "classnames": "^2.5.1",
     "npm-run-all": "^4.1.5"
+  },
+  "dependencies": {
+    "@mapbox/search-js-react": "^1.0.0-beta.19",
+    "prop-types": "^15.8.1",
+    "react": "^18.2.0"
```

Comment:
Are `react` and `prop-types` really dependencies or just dev dependencies? I don't see react anywhere on the front end, and I'd think we should be able to just use `wp.element` where we need react, unless the map requires a different version of react than the one in core.

Reply:
These are devDependancies, I'll update now.